### PR TITLE
Skyline/bugfix/20180912 small mol isotope dist hand edit fix

### PR DESCRIPTION
**Pull request by bspratt**
renewing this PR with a fix for a problem converting peptide docs to small molecule docs in tests

### CODE REVIEW

File: pwiz_tools/Skyline/Properties/Settings.cs [hunk 1]
```diff
@@ -2900,7 +2900,7 @@ public sealed class SrmSettingsList : SerializableSettingsList<SrmSettings>
                         new[] { IonType.y }, // PeptideFragmentTypes
                         new[] { Adduct.M_PLUS_H, }, // SmallMoleculePrecursorCharges
                         new[] { Adduct.M_PLUS, }, // SmallMoleculeProductCharges
-                        Transition.MOLECULE_ION_TYPES, // SmallMoleculeFragmentTypes
```
Comment (brendanx67) [inline pwiz_tools/Skyline/Properties/Settings.cs]: Let's not change the defaults in this pull request. Let's leave it compatible with the peptide defaults, and then we can implement a second change set that adds "p" to this setting when MS1 gets turned on. I don't think people doing SRM want this default precursor targeting, but everyone extracting MS1 chromatograms does, and we have a problem of teaching them to enable MS1 and add "p" to this setting. I don't think making "p" on by default only for small molecules is the solution.

Comment (bspratt) [reply pwiz_tools/Skyline/Properties/Settings.cs]: Fair enough, leaving it at default "f" makes it consistent with peptide behavior (though adding ", p" on enabling full scan would not be consistent with peptide behavior, so best to leave it alone completely I guess).

Comment (bspratt) [reply pwiz_tools/Skyline/Properties/Settings.cs]: OK, those changes are in.

File: pwiz_tools/Skyline/Model/RefinementSettings.cs [hunk 2]
```diff
@@ -1025,7 +1025,8 @@ public enum ConvertToSmallMoleculesChargesMode
                         Resources.RefinementSettings_ConvertToSmallMolecules_Converted_To_Small_Molecules,
```
Comment (brendanx67) [inline pwiz_tools/Skyline/Model/RefinementSettings.cs]: Let's go over this section. I still see a mix of "document" and "newDoc" which doesn't look completely rationalized to me.

**Pull request merged**